src/Microsoft.DotNet.Build.Tasks/PackageFiles/OptionalTooling.targets

```diff
@@ -66,7 +66,7 @@
           DependsOnTargets="CreateOptionalRestoreFeedItems;
                             PrepareOptionalToolProject">
     <PropertyGroup>
-      <GeneratedNuGetConfigDir>$(ObjDir)generatedNuGetConfig/</GeneratedNuGetConfigDir>
+      <GeneratedNuGetConfigDir>$(BaseIntermediateOutputPath)generatedNuGetConfig/</GeneratedNuGetConfigDir>
```

Comment:
Why did this one get changed from ObjDir to BaseIntermediateOutputPath?

Reply:
This is a temporary location for creating the nuget config and then using it which doesn't need to be the intermediate root. Correct me if I'm wrong.

Reply:
No it doesn't have to be in the root it just needs to be in the a place where the project will pick it up and we seem to be explicitly passing it to the restore command so I think the change is OK.